site/content/docs/cicd/settings.md

```diff
@@ -0,0 +1,35 @@
+# Deployment Settings File
+
+You can select deployment options interactively or specify them in a [JSON configuration file](docs/features/config-file.md). You can also keep the default values that the tool selects for you.
+
+This section defines the JSON definitions and syntax that you construct and use a deployment settings file. Here is the [JSON file definition](https://github.com/aws/aws-dotnet-deploy/tree/main/src/AWS.Deploy.Recipes/RecipeDefinitions).
+
+The Deployment Settings file is a JSON file that contains all of the settings that the deployment tool uses to drive the experience. The deployment tool uses this file to determine which deployment services to use, and how to configure them. --apply option on deploy command allows you to specify a deployment settings file.
+
+#### Creating a deployment settings file
+TODO
```

Comment:
Are we going to merge this PR with TODOs, or should this be as close as possible to the final pass?

Reply:
This can be merged with TODOs, which will need to be resolved before GA.